docs/troubleshooting/windows-pv-tools.md

```diff
@@ -60,4 +60,19 @@ It's possible that some antivirus blocks the end of the installation of the PV d
 
 ### Solution
 
-Simply pausing the agent and doing a reboot will make the XenTools to install succesfully. After a succesfull installation, enabling the SentinelOne agent again is possible without any other issues regarding the tools or drivers.
\ No newline at end of file
+Simply pausing the agent and doing a reboot will make the XenTools to install succesfully. After a succesfull installation, enabling the SentinelOne agent again is possible without any other issues regarding the tools or drivers.
+
+## Windows bug check 0x3B (SYSTEM_SERVICE_EXCEPTION) on systems with newer Intel CPUs
+
+### Cause
+
+Intel CPUs codenamed Rocket Lake, Sapphire Rapids and newer provide the **Architectural LBR** feature, which Windows depends on.
+Xen's support of this CPU feature is incomplete, which causes Windows to crash.
```

Comment:
This makes it looks like Windows doesn't work at all with these CPUs, but it's only when the feature is exerced, which it is not in many situations. Let's also mention the only actual case we know which is Blue Iris software (unless you know of other production software that makes the VM crash?), and reference the github issue and forum thread.

Reply:
Theoretically, anything that touches the last exception record (debuggers; error reporting) may crash the VM, but I've yet to confirm this.